frontends/PyCDE/test/good_example.py

```diff
@@ -6,7 +6,6 @@
 
 from pycde import dim, module, generator, types, Clock, Input, Output
 import pycde
-
```

Comment:
This appears to be unrelated. I also like to separate the std lib imports from the pycde/circt imports, just like we do in c++. If that's contrary to the pep8 style guide, then let's go with pep8.

Reply:
I don't know what the PEPs say about this, but I've used [isort](https://pycqa.github.io/isort/) to consistently handle std lib vs in tree vs local package.

Reply:
This was solely done to test whether the pr gate actually works... will be reverted once verified.